src/widgets/CardDescription/CardDescription.tsx

```diff
@@ -0,0 +1,28 @@
+import { useLoaderData } from 'react-router';
+import { Character } from '../../types';
+import { CardDetail } from './CardDetail';
+import './CardDescription.css';
+
+const CardDescription = (): JSX.Element => {
```

Comment:
```suggestion
const CardDescription: React.FC = () => {
```
indicate that you are using a functional component
do this in all components